app/controllers/anonymous_feedback/export_requests_controller.rb

```diff
@@ -33,11 +34,20 @@ def export_request_params
   end
 
   def anonymous_feedback_params
-    @anonymous_feedback_params ||= params.permit(:from, :to, :path, :organisation).to_h
+    clean_paths
+    @anonymous_feedback_params ||= params.permit(:from, :to, :organisation, :path, paths: []).to_h
+  end
+
+  def clean_paths
+    if params[:path]
+      params[:paths] = [params[:path]]
+    elsif params[:paths] && params[:paths].instance_of?(String)
+      params[:paths] = params[:paths].split(', ')
```

Comment:
The other clean_paths impl uses `split(',')` should we be consistent? FWIW I think the other one is "safer", we can always `split(',').map(&:strip)` if we're worried about leading or trailing spaces.